pxtpy/ast.ts

```diff
@@ -415,7 +415,7 @@ namespace pxt.py {
     }
     export interface NameConstant extends Expr {
         kind: "NameConstant";
-        value: boolean; // null=None, True, False
+        value: boolean | undefined; // undefined=None, True, False
```

Comment:
isn't this ``value?: boolean``

Reply:
I think the current version is clearer since undefined here represents a value